templates/poudriere.conf.erb

```diff
@@ -10,20 +10,20 @@
 # poudriere.
 #
 #ZPOOL=
-<% if @zpool and @zpool!='' -%>
+<% if @zpool -%>
```

Comment:
Does it make sense to switch over to epp and use `unless empty($zpool)`?

Reply:
I added switching from ERB to EPP in the modernization and cleanup issue